01_02_Combat_Rules.txt

```diff
@@ -681,7 +701,7 @@ sprint or use other movement abilities.
 
 	Going prone costs 1 Action, and standing up costs 1 Action. Lying flat down on
 your chest makes you a smaller target and helps you aim better, but also makes you
-easier to hit in melee. -1 gun miss chance while prone and -2 melee Guard DC. Being
+easier to hit in melee. -1 gun miss chance while prone and -2 melee Guard. Being
```

Comment:
sc: We should probably swap this to "+1 gun accuracy" for consistency sake.

Reply:
Scope creeping like a Zerg hive cluster 👍 🐢 
Will fix because those are friendly™️